tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2184,38 +2297,78 @@ public function userHasUnsharedASpaceSharedWith(
 		string $spaceName,
 		string $recipient
 	): void {
-		$this->sendUnshareSpaceRequest($user, $spaceName, $recipient);
+		$response = $this->sendUnshareSpaceRequest($user, $spaceName, $recipient);
 		$this->featureContext->theHTTPStatusCodeShouldBe(
 			200,
-			"Expected response status code should be 200"
+			"Expected response status code should be 200",
+			$response
 		);
 	}
 
 	/**
-	 * @When /^user "([^"]*)" unshares a space "([^"]*)" to (?:user|group) "([^"]*)"$/
-	 *
 	 * @param  string $user
 	 * @param  string $spaceName
 	 * @param  string $recipient
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
 	public function sendUnshareSpaceRequest(
 		string $user,
 		string $spaceName,
 		string $recipient
-	): void {
+	): ResponseInterface {
 		$space = $this->getSpaceByName($user, $spaceName);
 		$fullUrl = $this->baseUrl . $this->ocsApiUrl . "/" . $space['id'] . "?shareWith=" . $recipient;
 
+		return HttpRequestHelper::delete(
+			$fullUrl,
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user)
+		);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" unshares a space "([^"]*)" to (?:user|group) "([^"]*)"$/
+	 *
+	 * @param  string $user
+	 * @param  string $spaceName
+	 * @param  string $recipient
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userUnsharesSpace(
+		string $user,
+		string $spaceName,
+		string $recipient
+	): void {
 		$this->featureContext->setResponse(
-			HttpRequestHelper::delete(
-				$fullUrl,
-				$this->featureContext->getStepLineRef(),
-				$user,
-				$this->featureContext->getPasswordForUser($user)
-			)
+			$this->sendUnshareSpaceRequest($user, $spaceName, $recipient)
+		);
+	}
+
+	/**
+	 * @param  string $user
+	 * @param  string $object
+	 * @param  string $spaceName
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public function sendRemoveObjectFromSpaceRequest(
```

Comment:
and what is this object? file/folder?
```suggestion
	public function removeObjectFromSpace(
```

Reply:
both

Reply:
then `removeResourceFromSpace` would fit as the function name